packages/apps/dev-wallet/src/App/routes.tsx

```diff
@@ -47,6 +48,7 @@ export const Routes: FC = () => {
         <Route path="/" element={<HomePage />} />
         <Route path="/select-profile" element={<SelectProfile />} />
         <Route path="/create-profile" element={<CreateProfile />} />
+        <Route path="/personalize-profile" element={<PersonalizeProfile />} />
```

Comment:
Should it be a separate page/url or part of the `create-profile` @javadkh2?

Reply:
its better to to add it to the create-profile page. avoids code duplication